snapcraft/internal/lifecycle/_runner.py

```diff
@@ -59,6 +59,11 @@ def execute(step, project_options, part_names=None):
     :returns: A dict with the snap name, version, type and architectures.
     """
     config = project_loader.load_config(project_options)
+    if project_options.info.vendoring:
+        logger.warning(
+            'This snap uses vendoring but it cannot be enforced without '
+            'using cleanbuild. Proceeding without restriction.'.format(
+                ', '.join(project_options.info.vendoring)))
```

Comment:
This makes sense when running `snapcraft`, but when I run `snapcraft cleanbuild`, I still get it from the snapcraft instance running inside the container:

```
This snap uses vendoring. In order to enforce it, network access to resources not on the following list will be denied:

foo
[...]
This snap uses vendoring but it cannot be enforced without using cleanbuild. Proceeding without restriction.
```

We should probably clean that up.